database/models.py

```diff
@@ -6,6 +6,10 @@
 
 Base = declarative_base()
 
+class Update_Log(Base):
```

Comment:
@pradeepjairamani, please add a docstring for class detailing what is meant to do. And every class in models.py should have a ` __repr__(self)` function so that its representation is easier to see while debugging, please make one here.